test/kafo/configuration_test.rb

```diff
@@ -4,8 +4,8 @@
 module Kafo
   describe Configuration do
     let(:basic_config_file) { ConfigFileFactory.build('basic', BASIC_CONFIGURATION).path }
-    let(:basic_config) { Kafo::Configuration.new(basic_config_file, false) }
-    let(:old_config) { Kafo::Configuration.new(basic_config_file, false) }
+    let(:basic_config) { Kafo::Configuration.new(basic_config_file, Kafo::KafoConfigure.logger, false) }
```

Comment:
Perhaps a bit shorter in most places:
```suggestion
    let(:logger) { Kafo::KafoConfigure.logger }
    let(:basic_config) { Kafo::Configuration.new(basic_config_file, logger, false) }
```